test/parallel/test-module-children.js

```diff
@@ -0,0 +1,12 @@
+// Flags: --no-deprecation
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const dir = common.fixturesDir + '/GH-7131';
```

Comment:
I'm not sure since https://nodejs.org/api/modules.html#modules_all_together (module names are URL like)
Maybe `path.posix.join`, but I'm +1 on literal `/`